docs/_static/css/docsearch.overrides.css

```
@@ -28,3 +28,9 @@ a.algolia-docsearch-suggestion {
 body.theme-dark #search-form .algolia-autocomplete > .form-control {
   background: #1b1238 none repeat scroll 0% 0% padding-box !important;
 }
+
+/* overide algolia search default styles to prevent overflow on mobile devices */
+span#algolia-autocomplete-listbox-0 {
```

```suggestion
#algolia-autocomplete-listbox-0 {
```

No need to scope to the span here